internal/api/v1/user/configs/configs_api_test.go

```diff
@@ -60,7 +60,12 @@ func TestConfigsAPI_SharedConfig_APIResponses(t *testing.T) {
 
 			// then:
 			got, err := wallet.SharedConfig(context.Background())
-			require.ErrorIs(t, err, tc.expectedErr)
```

Comment:
Why not ErrorIs ? Doesn't work that way?

Reply:
with contains we have this 
```shell 
=== RUN   TestAccessKeyAPI_RevokeAccessKey
=== RUN   TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_str_response:_500
--- PASS: TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_str_response:_500 (0.01s)
=== RUN   TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_response:_200
--- PASS: TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_response:_200 (0.00s)
=== RUN   TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_response:_400
--- PASS: TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_response:_400 (0.00s)
--- PASS: TestAccessKeyAPI_RevokeAccessKey (0.02s)
PASS
ok      github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/user/users   1.062s
```
with errorIs we getting on same Test but with just the asserting change to ErrorIs
```shell
=== RUN   TestAccessKeyAPI_RevokeAccessKey
=== RUN   TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_response:_200
--- PASS: TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_response:_200 (0.01s)
=== RUN   TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_response:_400
--- PASS: TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_response:_400 (0.00s)
=== RUN   TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_str_response:_500
    /Users/augustynchmiel/4chain/spv/spv-wallet-go-client-v2/internal/api/v1/user/users/access_key_api_test.go:216:
                Error Trace:    /Users/augustynchmiel/4chain/spv/spv-wallet-go-client-v2/internal/api/v1/user/users/access_key_api_test.go:216
                Error:          Target error should be in err chain:
                                expected: "unrecognized response from API"
                                in chain: "failed to send HTTP DELETE request to revoke access key with ID: 081743f7-040e-45a3-8c36-dde39001e20d via User Access Keys API: HTTP response failure: unrecognized response from API: unexpected internal server failure"
                                        "HTTP response failure: unrecognized response from API: unexpected internal server failure"
                                        "unrecognized response from API: unexpected internal server failure"
                                        "unrecognized response from API"
                Test:           TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_str_response:_500
--- FAIL: TestAccessKeyAPI_RevokeAccessKey/HTTP_DELETE_/api/v1/users/current/keys/081743f7-040e-45a3-8c36-dde39001e20d_str_response:_500 (0.00s)
--- FAIL: TestAccessKeyAPI_RevokeAccessKey (0.02s)
FAIL
FAIL    github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/user/users   0.818s
```

Reply:
@chris-4chain if you prefer we could use also `require.ErrorContains(t, err, tc.expectedErr.Error())`